msteams-platform/task-modules-and-cards/what-are-cards.md

```diff
@@ -68,6 +68,26 @@ Type ahead search added as an input control in Adaptive Cards enable [dynamic se
 
 Adaptive Cards with Incoming Webhooks enables you to use the rich and flexible capabilities of Adaptive Cards. It sends data using Incoming Webhooks in Teams from their web service.
 
+### Adaptive cards form completion
+
+**Adaptive cards form completion** is a visual informational feedback message that appears when card action completes to a configurable set of card actions.
+Adaptive cards extensions are impacted with this feature
+
+User can see two types of feedback messages.
+
+**Error:** When a card action completes, user receives feedback as **Something went wrong, Try again** message if there is an error in the attempt.
+
+![Error message](~/assets/images/Cards/error-message.png)
+
+**Success:** When a card action completes, user receives feedback as **Your response was sent to the app** message if the attempt is successful.
+
+![Success message](~/assets/images/Cards/success-message.png)
+
+**Message as response type in mobile UX**
+Feedback message is shown at the bottom of the adaptive cards. User can click close button or switch chat to dismiss the message. There is no time logic for dismissal.
```

Comment:
The message appears at the bottom of the Adaptive Card. You can select **Close** or switch chat to dismiss the message.